### PR TITLE
Added Conflicts Page

### PR DESCRIPTION
**Pull request by SeqLaz**
This page describes when push changes create conflicts and how to resolve them.

### CODE REVIEW

File: documentation/reference/qfieldcloud/conflicts.en.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+title: Manage Conflicts
+tx_slug: documentation_reference_qfieldcloud_manage_conflicts
+---
+
+## Understanding Conflicts in Data Synchronization
+
+Conflicts occur when any of these conditions is met:
+
+1) distinct users set the same attribute on the same feature to different values, and the second patch of changes arrives before the first is applied to the project;
+2) a primary key is employed twice.
```
Comment (suricactus) [inline documentation/reference/qfieldcloud/conflicts.en.md:11]: Did we check this is true?

File: documentation/reference/qfieldcloud/conflicts.en.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Preventing, mitigating and resolving conflicts is crucial to maintaining data integrity in a healthy QGIS project. Here are some tips and tricks to do just that.
```
Comment (suricactus) [inline documentation/reference/qfieldcloud/conflicts.en.md:12]:
```suggestion
While highly unlikely have conflicts, preventing, mitigating and resolving conflicts is important to maintain data integrity in a healthy QGIS project. Here are some tips and tricks to do just that.
```
Makes the language a bit softer.

**Review by suricactus — COMMENTED** [review]
I think a screenshot/screencast showing "how to resolve deltas" would be super handy.

Did not check the English, but content wise it is correct.

File: documentation/reference/qfieldcloud/conflicts.en.md (hunk 1 above)
Comment (SeqLaz) [reply documentation/reference/qfieldcloud/conflicts.en.md:11]: @suricactus I can confirm this

File: documentation/reference/qfieldcloud/conflicts.en.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ Conflicts occur when any of these conditions is met:
 
 1) distinct users set the same attribute on the same feature to different values, and the second patch of changes arrives before the first is applied to the project;
 2) a primary key is employed twice.
-Preventing, mitigating and resolving conflicts is crucial to maintaining data integrity in a healthy QGIS project. Here are some tips and tricks to do just that.
+While highly unlikely have conflicts, preventing, mitigating and resolving conflicts is important to maintain data integrity in a healthy QGIS project. Here are some tips and tricks to do just that.
```
Comment (m-kuhn) [inline documentation/reference/qfieldcloud/conflicts.en.md:12]: This sentence still reads strange. What do we want to say with the first part?

Comment (SeqLaz) [reply documentation/reference/qfieldcloud/conflicts.en.md:12]: If the Overwrite conflicts is check but the second user pushed their changes and this is not applied this won't overwrite the conflicts

File: documentation/reference/qfieldcloud/conflicts.en.md [hunk 4]
```diff
@@ -25,9 +25,11 @@ While highly unlikely have conflicts, preventing, mitigating and resolving confl
...
-- When dealing with delta conflicts:
-   - Navigate to the DELTAS section.
-   - Filter the deltas with the "Not_applied" status.
-   - For each conflicted delta, select it and set the status to "Set status pending" from the "Action" dropdown menu, then press "Go."
-   - Check the details of changes in the conflict and click "APPLY DELTA" at the end of the page.
-   - Refer to our official documentation [Delta apply](https://docs.qfield.org/reference/qfieldcloud/jobs/#delta-apply-delta_apply-job) for additional guidance.
+- When dealing with conflicts:
+   - Navigate to the "Changes" section.
+   - Filter the changes with the "Not_applied" status.
+   - For each conflicted changes, select it and set the status to "Re-apply" from the "Action" dropdown menu, alternatively is all the new changes are in conflicts you can choose in the last conflicted change and select "Re-apply this and newer changes".
```
Comment (m-kuhn) [inline documentation/reference/qfieldcloud/conflicts.en.md:31]:
```suggestion
   - For each conflicted change, select it and set the status to "Re-apply" from the "Action" dropdown menu. Alternatively is all the new changes are in conflicts you can choose in the last conflicted change and select "Re-apply this and newer changes".
```
```suggestion
   - For each conflicted changes, select it and set the status to "Re-apply" from the "Action" dropdown menu, alternatively, if all the new changes are in conflict you can choose the last conflicted change and select "Re-apply this and newer changes".
```

Grammatically checked. Contents ok too, @suricactus ?

Comment (suricactus) [reply documentation/reference/qfieldcloud/conflicts.en.md]: Correct.

**Review by suricactus — APPROVED** [review]

File: documentation/reference/qfieldcloud/conflicts.en.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+   - For updating existing features based on field conditions, plan and designate the features each user will update.
+   - Users should avoid changing the "fid" or identifier numbers.
+
+### How to Resolve Conflicts?
+
+- By default, QFieldCloud overwrites conflicts using a _last wins_ policy (the latest patch of changes to the attribute(s) involved in the conflict replaces all earlier patches of changes to these attributes). Alternatively, admins can set a project's conflict resolution policy to _manual_. Doing so will require the project manager to manually resolve conflicts, picking those to be applied to the project.
+
+- When dealing with conflicts:
+   - Navigate to the "Changes" section.
+   - Filter the changes with the "Not_applied" status.
```
Comment (suricactus) [inline documentation/reference/qfieldcloud/conflicts.en.md:30]: @SeqLaz any chance you remmeber why it says "non_applied" here and not "conflict"?

Comment (SeqLaz) [reply documentation/reference/qfieldcloud/conflicts.en.md:30]: You are correct @suricactus  it should be with "Conflict" status.

Comment (SeqLaz) [reply documentation/reference/qfieldcloud/conflicts.en.md:30]: There the fix:
https://github.com/opengisch/QField-docs/pull/518

**Pull request merged**